### PR TITLE
created annoucement model, schema, and mutations in comms. Made a mig…

### PR DESCRIPTION
**Pull request by clam1987**
…ration file in comms

### CODE REVIEW

File: account/migrations/0041_merge_20200726_1740.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Generated by Django 3.0.3 on 2020-07-27 00:40
```
Comment (jerrylinew) [inline account/migrations/0041_merge_20200726_1740.py]: Can you fix these migration merge files? Rebase onto master, delete all the migration files you created (and these merge ones), drop your database, and then recreate your database and run makemigrations again.

File: comms/models.py [hunk 2]
```diff
@@ -7,11 +8,30 @@
...
 
+class Announcement(models.Model):
+    subject = models.TextField()
+    body = models.TextField()
+    course = models.ForeignKey(
+        Course,
+        on_delete=models.PROTECT
+    )
+    poster = models.ForeignKey(
+        get_user_model(),
```
Comment (jerrylinew) [inline comms/models.py:26]: If only instructors can post, perhaps make instructors the foreign key instead of user?

File: comms/mutations.py [hunk 3]
```diff
@@ -1,12 +1,72 @@
...
+            for enrollment in course.enrollment_set.all():
+                primary_parent = enrollment.student.primary_parent
+                email_address = primary_parent.user.email
+                primary_parent_fullname = primary_parent.user.first_name + ' ' +primary_parent.user.last_name
+                user = User.objects.get(id=validated_data.get('poster_id'))
+                poster = user.first_name + ' ' + user.last_name
+                email = Email.objects.create(
+                    template_id=ANNOUNCEMENT_EMAIL_TEMPLATE,
+                    recipient=email_address,
+                    data={'subject': validated_data.get('subject'), 'body': validated_data.get('body'), 'poster_name': poster, 'course': course.title, 'parent_name': primary_parent_fullname}
```
Comment (jerrylinew) [inline comms/mutations.py]: please format with one field of the dict per line

**Review by jerrylinew — CHANGES_REQUESTED** [review]
Great work! just some minor comments to address before I merge it in

File: comms/models.py (hunk 2 above)
Comment (clam1987) [reply comms/models.py:26]: I'm allowing instructors, admins, and receptionist to post that's why I used the get_user_model method.

**Review by jerrylinew — APPROVED** [review]
Looks great!

**Pull request merged**